### PR TITLE
DISTMYSQL-296 Release notes for PDPXC 8.0.32

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
new file:   docs/release-notes-pxc-v8.0.32.md
modified:   docs/release-notes.md
modified:   mkdocs-base.yml

### CODE REVIEW

File: docs/release-notes-pxc-v8.0.32.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+## Supplied components
+
+Review each component’s release notes for What’s new, improvements, or bug fixes. The following is a list of the components supplied with the Percona Server for MySQL-based variation of the Percona Distribution for MySQL:
+
+| Component               | Version   | Description                                |
+| ----------------------- | --------- | -------------------------------------------|
+| Percona XtraBackup      | [8.0.32-26](https://docs.percona.com/percona-xtrabackup/8.0/release-notes/8.0/8.0.32-26.0.html)| An open-source hot backup utility for MySQL-based servers that doesn’t lock your database during the backup.|
+| HAProxy                 | [2.6.12](https://git.haproxy.org/?p=haproxy-2.6.git;a=commit;h=f5884628ed25777fbffe9e3f097ffeff60e1fd1c) | A high-availability and load-balancing solution for Percona XtraDB Cluster. This is a default proxy.|
+| ProxySQL                | [2.5.1](https://docs.percona.com/proxysql/2.5.1.html)| A high performance, high-availability, protocol-aware proxy for MySQL.          |
```
Comment (eleo007) [inline docs/release-notes-pxc-v8.0.32.md:63]: If we release with 2.5.1 I would also add this as a separate note - that ProxySQL updated from 2.4 to 2.5

File: docs/release-notes-pxc-v8.0.32.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+* After a drop, adding a specific column using the INSTANT algorithm could cause a data error and a server exit. (Bug #34122122)
+
+* An online rebuild DDL no longer crashes after a column is added with `ALGORITHM=INSTANT`. Thank you Qingda Hu for reporting this bug. (Bug #33788578, Bug #106279)
+
+Find the full list of bug fixes and changes in the [MySQL 8.0.32 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-32.html).
+
+## Supplied components
+
+Review each component’s release notes for What’s new, improvements, or bug fixes. The following is a list of the components supplied with the Percona Server for MySQL-based variation of the Percona Distribution for MySQL:
```
Comment (kamil-holubicki) [inline docs/release-notes-pxc-v8.0.32.md]: There is no reference to the RN of PXC itself. Is it OK?

Comment (alina-derkach-oaza) [reply docs/release-notes-pxc-v8.0.32.md]: Yeah, we mention the PXC release notes at the beginning :

This release is focused on the Percona XtraDB Cluster-based deployment variation. It is based on [Percona XtraDB Cluster 8.0.32-24](https://www.percona.com/doc/percona-xtradb-cluster/8.0/release-notes/8.0.32-24.html)

File: docs/release-notes-pxc-v8.0.32.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/release-notes-pxc-v8.0.32.md:63]: added

File: docs/release-notes-pxc-v8.0.32.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
```
Comment (patrickbirch) [inline docs/release-notes-pxc-v8.0.32.md]: Should this say
Percona XtraDB Cluster-based variation

Comment (alina-derkach-oaza) [reply docs/release-notes-pxc-v8.0.32.md]: Updated

**Review by eleo007 — APPROVED** [review]

File: docs/release-notes-pxc-v8.0.32.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+| Component               | Version   | Description                                |
+| ----------------------- | --------- | -------------------------------------------|
+| Percona XtraBackup      | [8.0.32-26](https://docs.percona.com/percona-xtrabackup/8.0/release-notes/8.0/8.0.32-26.0.html)| An open-source hot backup utility for MySQL-based servers that doesn’t lock your database during the backup.|
+| HAProxy                 | [2.6.12](https://git.haproxy.org/?p=haproxy-2.6.git;a=commit;h=f5884628ed25777fbffe9e3f097ffeff60e1fd1c) | A high-availability and load-balancing solution for Percona XtraDB Cluster. This is a default proxy.|
+| ProxySQL                | [2.5.1](https://docs.percona.com/proxysql/2.5.1.html)| A high performance, high-availability, protocol-aware proxy for MySQL.          |
+| Percona Toolkit         | [3.5.2](https://www.percona.com/doc/percona-toolkit/LATEST/release_notes.html#v3-5-1-released-2022-11-28)     | The set of scripts to simplify and optimize database operation. |
```
Comment (patrickbirch) [inline docs/release-notes-pxc-v8.0.32.md]: Why does the link point to 3.5.1?
https://docs.percona.com/percona-toolkit/release_notes.html#v3-5-2-released-2023-03-28

Comment (alina-derkach-oaza) [reply docs/release-notes-pxc-v8.0.32.md]: done

**Review by patrickbirch — APPROVED** [review]
Good

**Pull request merged**